providers/aws/masterData.json

```diff
@@ -0,0 +1,2462 @@
+{
```

Comment:
Should this file still be here? would it be replaced by masterData.ftl?

Reply:
The current process to update the aws master data assumes a json file. So I left the masterdata.json there but the update script now creates an ftl file from the json file.